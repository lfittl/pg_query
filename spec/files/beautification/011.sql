SELECT a, b, max(c) FROM t group BY a,b having count(*) > 1 AND min(c) = 2
