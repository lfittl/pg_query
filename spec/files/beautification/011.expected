SELECT
  "a",
  "b",
  max("c")
FROM
  "t"
GROUP BY
  "a",
  "b"
HAVING
  count(*) > 1 AND
  min("c") = 2
